test/e2e/autotls/auto_tls_test.go

```diff
@@ -142,98 +116,44 @@ func disableNamespaceCert(t *testing.T, tlsClients *autoTLSClients) {
-	issuerRef := &cmmeta.ObjectReference{
-		Name: clusterIssuer.Name,
-		Kind: "ClusterIssuer",
-	}
-	issuerRefBytes, err := yaml.Marshal(issuerRef)
-	if err != nil {
-		t.Fatalf("Failed to convert IssuerRef %v to bytes: %v", issuerRef, err)
-	}
-
-	certManagerCM, err := tlsClients.clients.KubeClient.Kube.CoreV1().ConfigMaps(systemNamespace).Get("config-certmanager", metav1.GetOptions{})
-	if err != nil {
-		t.Fatalf("Failed to get config-certmanager ConfigMap: %v", err)
-	}
-	certManagerCM.Data["issuerRef"] = string(issuerRefBytes)
-	test.CleanupOnInterrupt(func() {
-		cleanUpConfigCertManagerCM(t, tlsClients)
-	})
-	if _, err := tlsClients.clients.KubeClient.Kube.CoreV1().ConfigMaps(certManagerCM.Namespace).Update(certManagerCM); err != nil {
-		t.Fatalf("Failed to update the config-certmanager ConfigMap: %v", err)
-	}
-	return func() {
-		cleanUpConfigCertManagerCM(t, tlsClients)
-	}
-}
-
-func cleanUpConfigCertManagerCM(t *testing.T, tlsClients *autoTLSClients) {
-	certManagerCM, err := tlsClients.clients.KubeClient.Kube.CoreV1().ConfigMaps(systemNamespace).Get("config-certmanager", metav1.GetOptions{})
-	if err != nil {
-		t.Errorf("Failed to get config-certmanager ConfigMap: %v", err)
-		return
-	}
-	delete(certManagerCM.Data, "issuerRef")
-	if _, err := tlsClients.clients.KubeClient.Kube.CoreV1().ConfigMaps(certManagerCM.Namespace).Update(certManagerCM); err != nil {
-		t.Errorf("Failed to clean up config-certmanager ConfigMap: %v", err)
-	}
-}
-
-func turnOnAutoTLS(t *testing.T, tlsClients *autoTLSClients) context.CancelFunc {
-	configNetworkCM, err := tlsClients.clients.KubeClient.Kube.CoreV1().ConfigMaps(systemNamespace).Get("config-network", metav1.GetOptions{})
+func turnOnAutoTLS(t *testing.T, clients *test.Clients) context.CancelFunc {
+	configNetworkCM, err := clients.KubeClient.Kube.CoreV1().ConfigMaps(systemNamespace).Get("config-network", metav1.GetOptions{})
 	if err != nil {
 		t.Fatalf("Failed to get config-network ConfigMap: %v", err)
 	}
 	configNetworkCM.Data["autoTLS"] = "Enabled"
 	test.CleanupOnInterrupt(func() {
-		turnOffAutoTLS(t, tlsClients)
+		turnOffAutoTLS(t, clients)
 	})
-	if _, err := tlsClients.clients.KubeClient.Kube.CoreV1().ConfigMaps(systemNamespace).Update(configNetworkCM); err != nil {
+	if _, err := clients.KubeClient.Kube.CoreV1().ConfigMaps(systemNamespace).Update(configNetworkCM); err != nil {
 		t.Fatalf("Failed to update config-network ConfigMap: %v", err)
 	}
 	return func() {
-		turnOffAutoTLS(t, tlsClients)
+		turnOffAutoTLS(t, clients)
 	}
 }
 
-func turnOffAutoTLS(t *testing.T, tlsClients *autoTLSClients) {
-	configNetworkCM, err := tlsClients.clients.KubeClient.Kube.CoreV1().ConfigMaps(systemNamespace).Get("config-network", metav1.GetOptions{})
+func turnOffAutoTLS(t *testing.T, clients *test.Clients) {
+	configNetworkCM, err := clients.KubeClient.Kube.CoreV1().ConfigMaps(systemNamespace).Get("config-network", metav1.GetOptions{})
```

Comment:
I am not sure that this will work in the `interrupt` case, .e.g if `t` is still setup at that point?

Reply:
I think we probably don't care about this case.